src/testE2E/codecatalyst/client.test.ts

```diff
@@ -354,16 +355,15 @@ describe.skip('Test how this codebase uses the CodeCatalyst API', function () {
             assert.ok(spaces.find(space => space.name === spaceName))
         })
 
-        it('lists all projects for the given user', async function () {
+        // TODO: Re-add this test and delete the projects after the test suite with the appropriate SDK API.
+        it.skip('lists all projects for the given user', async function () {
```

Comment:
Assuming the skip() is still intended, I don't quite understand the comment. What's meant by "the appropriate sdk api" ?

Reply:
updated this comment and one other one